pd-client/rpc_worker.go

```diff
@@ -35,10 +34,10 @@ type tsoRequest struct {
 	logical  int64
 }
 
-type regionRequest struct {
-	key    []byte
+type metaRequest struct {
+	pbReq  *pdpb.GetMetaRequest
 	done   chan error
-	region *metapb.Region
+	pbResp *pdpb.GetMetaResponse
```

Comment:
I think It is strange to put a response variable member in a struct.
Any suggestion? @siddontang